src/components/Stepper/MucStepperItem.vue

```diff
@@ -82,12 +95,17 @@ const getAriaLabel = computed(() =>
 );
 
 const handleClick = () => {
-  if (isDone) emit("click", item.id);
+  if (isDone && !disabled) emit("click", item.id);
```

Comment:
```suggestion
  if (isDone && !disabled) {
    emit("click", item.id);
  }
```
Even for a single line, we use brackets 😅